src/Elastic.Markdown/Assets/main.ts

```diff
@@ -1,10 +1,8 @@
-import {initNav} from "./pages-nav";
 import {initTocNav} from "./toc-nav";
 import {initHighlight} from "./hljs";
 import {initTabs} from "./tabs";
 import {initCopyButton} from "./copybutton";
 
-initNav();
```

Comment:
should this still be called? I don't see changes to `pages-nav.ts`

Reply:
Thank you 😅 

https://github.com/elastic/docs-builder/pull/533/commits/08424f79bdb39b11c2faa9f01d481b875dfd3933